src/Dockerfile/Arg.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Kiboko\Component\Dockerfile\Dockerfile;
+
+final readonly class Arg implements LayerInterface, \Stringable
+{
+    public function __construct(
+        private string $name,
+        private string $defaultValue,
```

Comment:
La valeur par défaut peut être absente
```suggestion
        private ?string $defaultValue = null,
```